core/src/main/java/org/elasticsearch/monitor/fs/FsInfo.java

```diff
@@ -506,6 +529,27 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         builder.field(Fields.TIMESTAMP, timestamp);
         builder.field(Fields.TOTAL);
         total().toXContent(builder, params);
+        if (leastDiskEstimate != null) {
+            builder.startObject(Fields.LEAST_ESTIMATE);
+            {
+                builder.field(Fields.PATH, leastDiskEstimate.getPath());
+                builder.byteSizeField(Fields.TOTAL_IN_BYTES, Fields.TOTAL, leastDiskEstimate.getTotalBytes());
+                builder.byteSizeField(Fields.AVAILABLE_IN_BYTES, Fields.AVAILABLE, leastDiskEstimate.getFreeBytes());
```

Comment:
I was initially confused why you put `freeBytes` into the `XContent` field `AVAILABLE_IN_BYTES`. Based on the example in the PR description, it makes sense to report this value as `AVAILABLE_IN_BYTES`. However, I wonder if it makes sense to add a new property `availableBytes` to `DiskUsage` which would make this clearer?